### DISCUSSION TITLE
Add Remote Site Path like IGNITION_REMOTE_SITES_PATH

### DISCUSSION THREAD

**Discussion by angus-mcritchie** [spatie/flareapp.io-roadmap#85] [Ideas]
File links don't work for remote container development, I use VS Code with WSL 2 and Docker.

With Ignition in Laravel, I can set my `.env` file with these to make it work. However, there's no way to set these values in the Flare portal.

```
IGNITION_EDITOR=vscode-remote`
IGNITION_REMOTE_SITES_PATH=/var/www/html
IGNITION_LOCAL_SITES_PATH=wsl+Ubuntu/home/user/projects/laravel/
```

Ideally, these values will be set at a user/project level, as different team members will likely have different configurations.

This improvement would help fix those pesky bugs even faster! 🙏

I just subscribed to Flare for production errors and love it. Keep up the great work ❤️

**Comment by mreduar** [comment]
Hello, have you found a solution to your problem? I am experiencing the same situation.
